packages/bundle/src/adaptiveCards/Attachment/AnimationCardContent.tsx

```diff
@@ -1,14 +1,14 @@
 /* eslint react/no-array-index-key: "off" */
 
-import { Components, hooks } from 'botframework-webchat-component';
+import { Components } from 'botframework-webchat-component';
 import PropTypes from 'prop-types';
 import React, { FC } from 'react';
 import type { DirectLineAnimationCard } from 'botframework-webchat-core';
 
 import CommonCard from './CommonCard';
+import useStyleSet from '../../hooks/useStyleSet';
```

Comment:
This `useStyleSet` contains style sets from our minimal bundle, plus from our full bundle (Adaptive Cards related style sets).